src/cljs/main/broadfcui/page/workspace/notebooks/notebooks.cljs

```diff
@@ -263,8 +277,17 @@
                    (fn [cluster]
                      (if (= (:status cluster) "Running")
                        (links/create-external
-                        {:href (str (config/leonardo-url-root) "/notebooks/" (:googleProject cluster) "/" (:clusterName cluster))
-                         :onClick #(utils/set-notebooks-access-token-cookie (utils/get-access-token))}
+                        {:href "javascript:;"
```

Comment:
Code changes in this file is meant to mirror:
https://github.com/DataBiosphere/leonardo/blob/develop/automation/src/test/resources/dummy-notebook-client.html

Reply:
This technique is problematic in that it will always cause the new window to be blocked by the user's browser.